runtime/src/bank_forks.rs

```diff
@@ -171,6 +171,7 @@ impl BankForks {
         let mut descendants = HashMap::<_, HashSet<_>>::new();
         for (slot, bank) in &banks {
```

Comment:
is banks ordered here or should we be maxing with the current `loaded_programs_cache.lastest_root`?

Reply:
Not sure they are ordered.
They should all have the same root, which is specified in the arguments. latest_root will always be 0 from the bank initialization. w/o this change, the only place its' updated is called from within `BankForks::set_root`, which we've certainly not called before we have `BankForks`